modules/installation-creating-aws-dns.adoc

```diff
@@ -6,12 +6,11 @@
 [id="installation-creating-aws-dns_{context}"]
 = Creating networking and load balancing components in AWS
 
-You must configure networking and load balancing (classic or network) in Amazon Web Services (AWS) for your
-{product-title} cluster to use. The easiest way to create these components is
-to modify the provided CloudFormation template, which also creates a hosted zone
-and subnet tags.
+You must configure networking and load balancing (classic or network) in Amazon Web Services (AWS) for your {product-title} cluster to use.
 
-You can run the template multiple times within a single VPC.
+You can use the provided CloudFormation template and a custom parameter file, to create a stack of AWS resources. The stack represents the networking and load balancing components required for your {product-title} cluster. The template is also used to create a hosted zone and subnet tags.
```

Comment:
See previous comment about "file, to"

For active voice, if possible:
- s/required for your {product-title} cluster/that your {product-title} cluster requires
- s/The template is also used to create a hosted zone and subnet tags./The template also creates a hosted zone and subnet tags./

Reply:
Thank you for these suggestions, these make sense to me. Resolved.